account_tax_balance/tests/test_account_tax_balance.py

```diff
@@ -108,8 +108,8 @@ def test_tax_balance(self):
 
         # exercise search has_moves = True
         taxes = self.env['account.tax'].search([('has_moves', '=', True)])
-        self.assertEqual(len(taxes), 1)
-        self.assertEqual(taxes[0].name, "Tax 10.0%")
+        self.assertLessEqual(tax, taxes)
+        self.assertEqual(tax.name, "Tax 10.0%")
```

Comment:
This test is useless.

Reply:
That's what I thought. Let me remove it.